config_custom/control.ini

```diff
@@ -1,23 +1,9 @@
 [Control]
 exclude = pretest_example,intratest_example,posttest_example,example,subexample
-          ,docker_cli/build/git_path
-          ,docker_cli/build/dockerignore
-          ,docker_cli/cp
-          ,docker_cli/info
-          ,docker_cli/build
-          ,docker_cli/deferred_deletion
-          ,docker_cli/events
-          ,docker_cli/images_all
-          ,docker_cli/liverestore
-          ,docker_cli/load
-          ,docker_cli/login
-          ,docker_cli/pull
-          ,docker_cli/run_attach
-          ,docker_cli/run_cgroups
-          ,docker_cli/run_exec/exec_false
-          ,docker_cli/run_exec/exec_pid_count
```

Comment:
exec_pid_count has just been fixed, next build should include it. All the rest of run_exec passes; I think run_exec should be included.